floodwatch/src/js/components/Compare.js

```diff
@@ -62,43 +86,84 @@ export class CompareContainer extends Component {
 
   async componentDidMount(): any {
     const AdBreakdown = await FWApiClient.get().getFilteredAdCounts({ filterA: {}, filterB: {} })
+    const FilterATopic = d3.entries(AdBreakdown.filterA.categories).sort(function(a, b) {
+      return d3.descending(a.value, b.value);
+    })[0]
+
+    const UserData = await FWApiClient.get().getCurrentPerson()
 
     this.setState({
       leftData: AdBreakdown.filterA.categories,
-      rightData: AdBreakdown.filterB.categories
+      rightData: AdBreakdown.filterB.categories,
+      currentTopic: FilterATopic.key,
+      userData: UserData
+    })
+  }
+
+  updateMouseOver(newTopic: string): void {
+    this.setState({
+      currentTopic: newTopic
     })
   }
 
-  createSentence(options: Object): string {
-    let sentence = 'Floodwatch users';
-    if (options.length == 0) {
-      sentence = 'All ' + sentence;
-      return sentence
+  changeCategoriesCustom(side: string, mouse: Event, info: Filter, checked: boolean, event: any): void {
```

Comment:
Also I'm pretty sure you bound `this` to the far inner component... is that the intention? There's probably a cleaner way to get the options of that component. Will ask you.